src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -41,7 +41,8 @@ public record CreateCollectionOperation(
     String vectorFunction,
     String comment,
     int ddlDelayMillis,
-    boolean tooManyIndexesRollbackEnabled)
+    boolean tooManyIndexesRollbackEnabled,
+    boolean denyAll)
```

Comment:
This may be the way to do it but it'd be great if we did not have this many arguments and `CollectionSettings` were passed as object.

But for readability maybe rename as `indexingDenyAll`?